### PR TITLE
Fix dead repository link to Java example

### PR DESCRIPTION
**Pull request by Riggs333**

### CODE REVIEW

**Comment by Riggs333** [comment]
Sorry, I somehow missed seeing ticket #342 before creating the PR.
Feel free to reject this PR.

**Review by mlvandijk — CHANGES_REQUESTED** [review]
Could you update it in line with #342 please?

**Comment by luke-hill** [comment]
@Riggs333 are you able to do this?

File: content/docs/tools/java.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ To run Cucumber with [Gradle](https://gradle.org/), make sure that:
 
 Steps:
 
-1.  Create a new Gradle project or look at [java-gradle](https://github.com/cucumber/cucumber-jvm/tree/master/examples/java-gradle) example on Github
+1.  Create a new Gradle project or look at [cucumber-java-skeleton](https://github.com/cucumber/cucumber-java-skeleton) example on Github
```
Comment (mlvandijk) [inline content/docs/tools/java.md]:
```suggestion
1.  Create a new Gradle project or look at [gradle-cucumber-runner](https://github.com/tsundberg/gradle-cucumber-runner) example on Github
```

**Review by mlvandijk — COMMENTED** [review]
Added suggestion with link from #355 342

File: content/docs/tools/java.md (hunk 1 above)
Comment (mlvandijk) [reply content/docs/tools/java.md]: @Riggs333 Are you able to commit the suggestion so we can merge your PR?

**Comment by Riggs333** [comment]
I'm very sorry for being this late to answer here and for applying the requested change.
@mlvandijk Would you please take a look if this is ok now?

**Review by mlvandijk — APPROVED** [review]

**Comment by mlvandijk** [comment]
Thanks @Riggs333 !

**Pull request merged**